components/facebook_groups/README.md

```diff
@@ -0,0 +1,7 @@
+# Getting Started
+In order to connect your Facebook Group to Pipedream, you'll need to navigate to the group which you are an administrator of, and in the lefthand sidebar, under **Settings** > **Group Settings**, you will find **Manage Advanced Settings**. Click the edit button under **Apps**.
+![Apps](https://res.cloudinary.com/dpenc2lit/image/upload/v1687367319/mycology_fb_bpfmh3.png)
+
+Search for "Pipedream", and add the app to your Facebook Group. Once the Pipedream app has successfully been added to your group, you can start building automated workflows on Pipedream using Facebook Groups.
```

Comment:
```suggestion
Search for "Pipedream", and add the app to your Facebook Group. Once the Pipedream app has been successfully added to your group, you can start building automated workflows on Pipedream using Facebook Groups.
```

Reply:
Changed 'has successfully been added' to 'has been successfully added' (Dreyer's English: clarity)